OracleTSAM/samples/xe-db-svc/README.md

```diff
@@ -0,0 +1,42 @@
+Running Oracle TSAM Plus with oracle/database:11.2.0.2-xe image
+=============================
+You can run the TSAM container with the `oracle/database:11.2.0.2-xe` image provided on [GitHub](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+
+Due to the `oracle/database:11.2.0.2-xe` container creates new database and generates random `sys` user password on startup, a little "hacking" is needed here to make the `sys` password fixed.
+
+Basically this is done by overriding the default command `runOracle.sh` with this [modified one](cmd/runOracle.sh) in the `cmd` folder. The `docker-compose.yml` file to start the containers will look like below:
```

Comment:
The password is randomly generated as per policy. This was discuss with @Djelibeybi before. I don't think it makes sense to provide a copy of the run script as I can't see how it will stay in sync with the actual image. The default password can be easily overwritten with the `setPassword.sh` script inside the DB container without having to know the generated one. I highly advise to use that script if possible.